R/makeTreeSummarizedExperimentFromqiime2.R

```diff
@@ -114,15 +156,18 @@ makeTreeSummarizedExperimentFromqiime2 <- function(featureTableFile,
 #' @importFrom ape read.tree
 #' @importFrom Biostrings readDNAStringSet
 #' @keywords internal
-.read_qza <- function(file, temp = tempdir()) {
+.read_qza <- function(file, temp = tempdir(), ...) {
```

Comment:
the `...` argument does not used in the function body

Reply:
It is required, since the `makeTreeSummarizedExperimentFromqiime2` uses it for two functions. If you would remove it, an error would be thrown in case both `temp` and `removeTaxaPrefix` would be set by the user.